DOC.md

```diff
@@ -1399,7 +1399,7 @@ If `jsregexp` is not available, transformation are replaced by a simple copy.
 # VARIABLES
 
 All `TM_something`-variables are supported with two additions:
-`SELECT_RAW` and `SELECT_DEDENT`. These were introduced because
+`LS_SELECT_RAW` and `LS_SELECT_DEDENT`. These were introduced because
```

Comment:
Ah, could you add the new vars here too?
Best make it a list for readability

Reply:
I think we should (not in this PR) list the TM_ ones too.

Reply:
A link is ok for the online doc, but for the .txt I think it's better to have it in place :thinking:

Reply:
Hmm yes, good point... I don't think `panvimdoc` allows .txt-specific sections though.
OTOH, just making a big list should be okay for the markdown as well, then we could even mention details of our implementation.